flink-ml-lib/src/test/java/org/apache/flink/ml/evaluation/binaryeval/BinaryClassificationEvaluatorTest.java

```diff
@@ -0,0 +1,211 @@
+                    Row.of(0.0, Vectors.dense(0.8, 0.2)),
+                    Row.of(1.0, Vectors.dense(0.9, 0.1)));
+
+    private static final List<Row> INPUT_DATA_WITH_WEIGHT =
+            Arrays.asList(
+                    Row.of(1.0, Vectors.dense(0.1, 0.9), 0.8),
+                    Row.of(1.0, Vectors.dense(0.1, 0.9), 0.7),
+                    Row.of(1.0, Vectors.dense(0.1, 0.9), 0.5),
+                    Row.of(0.0, Vectors.dense(0.25, 0.75), 1.2),
+                    Row.of(0.0, Vectors.dense(0.4, 0.6), 1.3),
+                    Row.of(1.0, Vectors.dense(0.1, 0.9), 1.5),
+                    Row.of(1.0, Vectors.dense(0.1, 0.9), 1.4),
+                    Row.of(0.0, Vectors.dense(0.6, 0.4), 0.3),
+                    Row.of(0.0, Vectors.dense(0.7, 0.3), 0.5),
+                    Row.of(1.0, Vectors.dense(0.1, 0.9), 1.9),
+                    Row.of(0.0, Vectors.dense(0.8, 0.2), 1.2),
+                    Row.of(1.0, Vectors.dense(0.9, 0.1), 1.0));
+
+    private static final double[] EXPECTED_DATA =
+            new double[] {0.7691481137909708, 0.3714285714285714, 0.6571428571428571};
+    private static final double[] EXPECTED_DATA_M =
+            new double[] {0.8571428571428571, 0.9377705627705628, 0.8571428571428571};
+    private static final double EXPECTED_DATA_W = 0.8911680911680911;
+    private static final double EPS = 1.0e-5;
+
+    @Before
+    public void before() {
+        Configuration config = new Configuration();
+        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(3);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        tEnv = StreamTableEnvironment.create(env);
+        inputDataTable =
+                tEnv.fromDataStream(env.fromCollection(INPUT_DATA)).as("label", "rawPrediction");
+        inputDataTableWithMultiScore =
+                tEnv.fromDataStream(env.fromCollection(INPUT_DATA_WITH_MULTI_SCORE))
+                        .as("label", "rawPrediction");
+        inputDataTableWithWeight =
+                tEnv.fromDataStream(env.fromCollection(INPUT_DATA_WITH_WEIGHT))
+                        .as("label", "rawPrediction", "weight");
+    }
+
+    @Test
+    public void testParam() {
+        BinaryClassificationEvaluator binaryEval = new BinaryClassificationEvaluator();
+        assertEquals("label", binaryEval.getLabelCol());
+        assertNull(binaryEval.getWeightCol());
+        assertEquals("rawPrediction", binaryEval.getRawPredictionCol());
+        assertArrayEquals(
+                new String[] {"areaUnderROC", "areaUnderPR"}, binaryEval.getMetricsNames());
+        binaryEval
+                .setLabelCol("labelCol")
+                .setRawPredictionCol("raw")
+                .setMetricsNames("areaUnderROC")
```

Comment:
nit: It might be better to use the constants we have already defined. For example, `setMetricsNames(BinaryClassificationEvaluatorParams.AREA_UNDER_ROC)`